### PR TITLE
[v1] more 1.x updates... (import)

### PR DESCRIPTION
**Pull request by jorgeorpinel**
- A typo fix
- Review of `import[-url]` triggered by https://github.com/iterative/dvc/issues/4520#issuecomment-759617964

### CODE REVIEW

**Comment by jorgeorpinel** [comment]
Notice this is a PR into `v1` — sorry, I decided to do one more of these since these changes fully apply to 1.x.

Will need a fwdport PR into `master` once this is merged.

**Review by shcheklein — APPROVED** [review]

**Comment by jorgeorpinel** [comment]
> Notice this is a PR into v1

Wait I merged this to master haha. That's OK, the branch cam from `v1` so, same result. I'll just backport this into `v1` then.

File: content/docs/command-reference/import-url.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # import-url
 
 Download a file or directory from a supported URL (for example `s3://`,
-`ssh://`, and other protocols) into the <abbr>workspace</abbr>, and track
-changes in the remote data source. Creates a `.dvc` file.
+`ssh://`, and other protocols) into the <abbr>workspace</abbr>, and track it (an
+import `.dvc` file is created).
```
Comment (jorgeorpinel) [inline content/docs/command-reference/import-url.md:5]: For `import-url -h`

Comment (jorgeorpinel) [inline content/docs/command-reference/import-url.md:5]: For `import-url -h`

File: content/docs/command-reference/import.md [hunk 2]
```diff
@@ -1,9 +1,7 @@
 # import
 
-Download a file or directory tracked by DVC or by Git into the
-<abbr>workspace</abbr>. It also creates a `.dvc` file with information about the
-data source, which can later be used to [update](/doc/command-reference/update)
-the import.
+Download a file or directory tracked by another DVC or Git repository into the
+<abbr>workspace</abbr>, and track it (an import `.dvc` file is created).
```
Comment (jorgeorpinel) [inline content/docs/command-reference/import.md:4]: For `import -h`

**Review by jorgeorpinel — COMMENTED** [review]
Check corresponding texts in core repo:

**Pull request merged**